src/python/pants/backend/docker/util_rules/docker_build_context.py

```diff
@@ -283,6 +283,7 @@ async def create_docker_build_context(
             UnparsedAddressInputs(
                 dockerfile_build_args.values(),
                 owning_address=dockerfile_info.address,
+                description_of_origin="TODO(#14468)",
```

Comment:
I can try to figure this out, although @kaos if you have a chance, it would help.